winhttp_data_windows.go

```diff
@@ -0,0 +1,42 @@
+package ieproxy
```

Comment:
Given that we now have the winHttp DLL & Procs in here now, I think `winhttp_data_windows.go` is still valid, since it contains a variety of information about winHttp.

Reply:
Yeah, OK.